src/systemd_ctypes/event.py

```diff
@@ -16,19 +16,25 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import asyncio
+import logging
 import selectors
 import sys
 from typing import Callable, ClassVar, Coroutine, List, Optional, Tuple
 
 from . import inotify, libsystemd
 from .librarywrapper import Reference, UserData, byref
 
+logger = logging.getLogger(__name__)
+
+MAX_UINT64 = 1 << 64 - 1
```

Comment:
In Python minus binds stronger than `<<` so you end up with `1 << 63` here. I don't think that's what you intend?
```
>>> 1 << 63
9223372036854775808
>>> f"{1 << 63:b}"
'1000000000000000000000000000000000000000000000000000000000000000'
```
but `(1 << 63) - 1` which is equal to `sys.maxsize` on 64bit systems